functions/backup.bash

```diff
@@ -44,15 +42,15 @@ restore_openhab_config() {
   fi
 
   local backupList
-  local backupPath="/var/lib/openhab2/backups"
+  local backupPath="${OPENHAB_BACKUPS:-/var/lib/openhab2/backups}"
```

Comment:
Where does this come from, meaning `$OPENHAB_BACKUPS`

Reply:
/etc/default/openhab[2]
but only if manually specified there which it isn't by default.